spyder/locale/es/LC_MESSAGES/spyder.po

```diff
@@ -1532,7 +1544,7 @@ msgstr "Mostrar contenidos de listas"
 
 #: spyder/plugins/completion/languageserver/confpage.py:723
 msgid "Show automatic completions after characters entered:"
-msgstr ""
+msgstr "Mostrar completado automático después de ingresar los caracteres:"
```

Comment:
"Número de caractéres mínimos para mostrar completado automáticamente?"

Reply:
Agree. I'll fix it in the next commit.